src/kernel-install/60-ukify.install.in

```diff
@@ -210,22 +211,30 @@ def initrd_list(opts) -> list[Path]:
     return [*microcode, *opts.initrd, *initrd]
 
 
-def call_ukify(opts):
+def load_module(name: str, path: str) -> types.ModuleType:
     # Punish me harder.
     # We want this:
     #   ukify = importlib.machinery.SourceFileLoader('ukify', UKIFY).load_module()
     # but it throws a DeprecationWarning.
     # https://stackoverflow.com/questions/67631/how-can-i-import-a-module-dynamically-given-the-full-path
     # https://github.com/python/cpython/issues/65635
     # offer "explanations", but to actually load a python file without a .py extension,
-    # the "solution" is 4+ incomprehensible lines.
-    # The solution with runpy gives a dictionary, which isn't great, but will do.
-    ukify = runpy.run_path(UKIFY, run_name='ukify')
+    # the "solution" is 4 incomprehensible lines below.
+    # runpy.run_path() is easier, but gives a dictionary, not a module.
+    loader = importlib.machinery.SourceFileLoader(name, path)
+    spec = importlib.util.spec_from_file_location(name, path, loader=loader)
```

Comment:
```suggestion
    spec = importlib.util.spec_from_file_location(name, path)
```
I think specifying the loader can be dropped. Looking at the cpython code it will try out loaders for extension modules, source files and byte codes in order, so it will use this one anyway.

Reply:
When I wrote "incomprehensible", I wasn't joking.

Reply:
Ah, that explains, why it worked when I tested it against the git tree. There it if course has the proper extension. :) Maybe add a comment about this, since this extends the linked things.